socorro/schemas/README.rst

```diff
@@ -140,11 +140,11 @@ Field details:
    .. code-block:: yaml
 
       examples:
-        - "`Auto`"
-        - "`Infobar`"
-        - "`AboutCrashes`"
-        - "`CrashedTab`"
-        - "`Client`"
+        - "Auto"
+        - "Infobar"
+        - "AboutCrashes"
+        - "CrashedTab"
+        - "Client"
```

Comment:
I moved the markdown bit to the template so it's easier to add examples in the schema.